### PR TITLE
Creates github-issue helper + test. Adds links to demo app at each test section.

### PR DESCRIPTION
**Pull request by thedig**
In response to issue here:  https://github.com/ember-a11y/ember-a11y/issues/7

The links are added manually to the demo app at each test section. It looks like there's opportunity to create a component for the link in addition to the helper. However I then had questions of if that was more in the scope of looking at making a more general component for each of the test sections that have (mostly) the same steps listed.

### CODE REVIEW

File: addon/helpers/github-issue.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function githubIssue([title]/*, hash*/) {
+	let titleStr = encodeURIComponent(`Demo App Test fail: ${title}`);
+	let linkStr = 
+		`https://github.com/nathanhammond/ember-a11y/issues/new?title=${titleStr}`;
+  return linkStr;
```
Comment (nathanhammond) [inline addon/helpers/github-issue.js]: Did you know that you can return a DOM node from a helper? `document.createElement('a')` works here. :) You can use that to move more of the markup into the helper.

File: tests/dummy/app/templates/application.hbs [hunk 2]
```diff
@@ -44,6 +49,10 @@
       <li>Content should include "Global Error".</li>
       <li>No focus should have been set.</li>
     </ol>
+    <p>
+      Did this test fail? Let us know 
+        <a href={{github-issue "Error, Sibling Error Route, Top Level"}}>here</a>.
+    </p>
```
Comment (nathanhammond) [inline tests/dummy/app/templates/application.hbs]: All of this markup could be pulled into the helper.

File: addon/helpers/github-issue.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+}
+
+export default Ember.Helper.helper(githubIssue);
```
Comment (nathanhammond) [inline addon/helpers/github-issue.js]: We should move this entire helper into the test application instead of in the top-level addon.

**Comment by nathanhammond** [comment]
This looks great! I'm excited to help you get the last few pieces done to get this landed. I've left a few line notes and you can find me on Slack to answer any questions you have!

File: addon/helpers/github-issue.js (hunk 1 above)
Comment (thedig) [inline addon/helpers/github-issue.js]: oh that's great!

**Comment by binhums** [comment]
@nathanhammond looks like your comments have been addressed, are you happy to merge?

@thedig :+1: Thanks for putting this together. It's worth adding a comment after responding to code review, github won't notify anyone on new commits.

For the record I'm totally against writing HTML outside of template files. If we're using a helper to signify HTML we should really just use a component.

**Comment by nathanhammond** [comment]
Let's land as is, and then convert into a helper. :)

**Pull request merged**